### DISCUSSION TITLE
Unable to insert into table when connected via CLI

### DISCUSSION THREAD

**Discussion by dextermb** [planetscale/discussion#144] [Q&A] [answered]
Hey all

I'm trying to insert some data into a table via `pscale` but I'm running into an error.

```
db/main> insert into Org(name) values('Acme');
ERROR 1364 (HY000): target: db.-.primary: vttablet: rpc error: code = Unknown desc = (errno 1364) (sqlstate HY000) (CallerID: unsecure_grpc_client): Sql: "insert into Org(`name`) values (:vtg1)", BindVars: {}
```

Edit:

Thought it might be an issue with my CLI setup but it seems like Arctype is having the same issues:
<img width="492" alt="image" src="https://user-images.githubusercontent.com/6137789/146376146-75dccdfd-9a3a-4329-b200-b0c6794c1560.png">

---

Any ideas why I might be getting this?

**Comment by jgreet** [comment, marked as answer]
Sorry, that error message really should be more helpful. I'll bring this post up with engineering.

I took a look at your branch's schema. The issues is likely that your Org table has an additional field `id` that is set to `NOT NULL`, and that doesn't have a default value defined, making it required for inserts like this.

Try adding a value for `id` as well:

```sql
 insert into Org (id, name) values('xyz123', 'Acme');
```

**Reply by dextermb** [reply]
Thanks John, I'll give this a shot. Within my Prisma schema it shows that `Org.id` has a default value but I suppose that's just on the prisma side of things.

```prisma
model Org {
  id String @id @default(cuid())
  name String?

  admins OrgAdmin[]
}
```

And in terms of the migration generated by Prisma I can confirm that the default value is generated in runtime.
```sql
-- CreateTable
CREATE TABLE `Org` (
    `id` VARCHAR(191) NOT NULL,
    `name` VARCHAR(191) NULL,

    PRIMARY KEY (`id`)
) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
```

I should of checked here but the error completely threw me off!
